aspnetcore/getting-started.md

```diff
@@ -17,28 +17,61 @@ uid: getting-started
 
 1. Install the [!INCLUDE[](~/includes/2.1-SDK.md)].
 
-2. Create a new .NET Core project.
+2. Create an ASP.NET Core project. Open a command shell and enter the following command:
 
-   On macOS and Linux, open a terminal window. On Windows, open a command prompt. Enter the following command:
-
-    ```terminal
+    ```console
     dotnet new webapp -o aspnetcoreapp
     ```
 
-3. Run the app with the following commands:
+3. Trust the HTTPS development certificate:
+
+# [Windows](#tab/windows)
+
+    ```console
+    dotnet dev-certs https --trust
+    ```
+
+    The preceding command displays the following dialog:
+
+    ![Security warning dialog](getting-started/_static/cert.png)
+
+    Select **Yes** if you agree to trust the development certificate.
+
+# [macOS](#tab/macos)
+
+    ```console
+    dotnet dev-certs https --trust
+    ```
+
+    The preceding command displays the following message:
 
-    ```terminal
+    *Trusting the HTTPS development certificate was requested. If the certificate is not already trusted we will run the following command:*
+    `'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'`
+    *This command might prompt you for your password to install the certificate on the system keychain.
+    Password:*
+
+    Enter your password if you agree to trust the development certificate.
+
+# [Linux](#tab/linux)
+
+    See the Linux platform documentation.
```

Comment:
How about this: "See the documentation for your Linux distribution on how to trust the HTTPS development certificate"